code/game/objects/structures/salvaging.dm

```diff
@@ -210,14 +230,14 @@
 		if(41 to 60)
 			visible_message("<span class='danger'>You flinch as the [src]'s laser apparatus lights up, but your tool destroys it before it activates...</span>")
 		if(61 to 79)
-			visible_message("<span class='danger'>You see a bright light from the [src] before the laser reactivates in your face!</span>")
+			visible_message("<span class='danger'>You see a dim light from the [src] before the laser reactivates in your face!</span>")
 			shoot_projectile(user, /obj/projectile/beam/scatter)
 		if(80 to 89)
 			visible_message("<span class='danger'>You see a bright light from the [src] before the laser reactivates in your face!</span>")
 			shoot_projectile(user, /obj/projectile/beam)
 		if(90 to 100)
-			visible_message("<span class='danger'>You see a bright light from the [src] before the laser reactivates in your face!</span>")
-			shoot_projectile(user, /obj/projectile/beam/laser/heavylaser)
+			visible_message("<span class='danger'>You see an intense light from the [src] before the laser reactivates in your face!</span>")
+			shoot_projectile(user, /obj/projectile/beam/laser/heavylaser) //i'd like to make this flash people. but i'm not sure how to do that. shame!
```

Comment:
just gotta make a new projectile type that calls flash_act() on the target in its on_hit() proc